warehouse/slave/worker.go

```diff
@@ -33,6 +34,7 @@ import (
 var (
 	errIncompatibleSchemaConversion = errors.New("incompatible schema conversion")
 	errSchemaConversionNotSupported = errors.New("schema conversion not supported")
+	json                            = jsoniter.ConfigCompatibleWithStandardLibrary
```

Comment:
We can even explore `jsoniter.ConfigFastet` as well.  https://github.com/json-iterator/go/wiki/Config#built-in-configuration
```suggestion
	json                            = jsoniter.ConfigFastest
```